### PR TITLE
fix: Updated default template names with 'learner-record'

### PR DESCRIPTION
**Pull request by mlscandinaro**
[mscandinaro/MICROBA-1355]

fix: changed default template names

docs: added documentation to README

### CODE REVIEW

**Review by staubina — APPROVED** [review]
Looks good to me.

File: README.rst [hunk 1]
```diff
@@ -1,55 +1,49 @@
...
-In the following steps, replace "frontend-template-application' with the name of the repo you created when copying this template above.
-
-1. Clone your new repo:
-
-  ``git clone https://github.com/edx/frontend-template-application.git``
+Installation
+------------
 
-2. Use node v12.x.
+This MFE is bundled with `Devstack <https://github.com/edx/devstack>`_, see the `Getting Started <https://github.com/edx/devstack#getting-started>`_ section for setup instructions.
```
Comment (justinhynes) [inline README.rst:18]: I'm not sure if this statement is true? I don't think this is currently bundled with devstack?

I think we would provide instructions to pull, build, and start the Learner Record MFE here.

File: public/index.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <!doctype html>
 <html lang="en-us">
   <head>
-      <title>Application Template | <%= process.env.SITE_NAME %></title>
+      <title>My Learner Records | <%= process.env.SITE_NAME %></title>
```
Comment (justinhynes) [inline public/index.html:4]: I am wondering if the `My Learner Records` text is something we would translate in other MFEs. Might be worth checking some of the other MFE repos to confirm?

Comment (mlscandinaro) [reply public/index.html:4]: Ok, I did not see translations for the titles in other MFEs.

File: README.rst (hunk 1 above)
Comment (mlscandinaro) [reply README.rst]: I don't think it's true either. I updated the instructions so it's more focused on the MFE.

**Review by justinhynes — APPROVED** [review]

**Pull request merged**